doc/main.html

```diff
@@ -5,22 +5,30 @@
     <meta http-equiv="x-ua-compatible" content="ie=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <script type="module" src="../tooling/smpte.js"></script>
+
+
     <meta itemprop="pubType" content="ST" />
-    <meta itemprop="pubNumber" content="999" />
-    <!-- <meta itemprop="pubPart" content="99" /> -->
+    <meta itemprop="pubNumber" content="100" />
+    <meta itemprop="pubPart" content="10" />
     <meta itemprop="pubState" content="draft" />
     <meta itemprop="pubStage" content="WD" />
-    <meta itemprop="pubTC" content="27C" />
-    <meta itemprop="pubDateTime" content="2024-01-01" />
+    <meta itemprop="pubConfidential" content="yes" />
+    <meta itemprop="pubTC" content="XX" />
+    <meta itemprop="pubSuiteTitle" content="HTML Pub Tooling" />
+    <meta itemprop="pubDateTime" content="2024-09-30" />
     <!-- <meta itemprop="pubRevisionOf" content="SMPTE ST XXXX-Y:ZZZZ" /> -->
-    <!-- <meta itemprop="pubSuiteTitle" content="Sample Document" /> -->
-    <title>Title of the document</title>
+
+    <title>Template</title>
```

Comment:
Do you think this is clearer?

Reply:
Yes, so it's more like a document, when used in conjunction with the guide. But also so we can clearly see that it's the template doc.